src/cascadia/TerminalApp/ActionArgs.idl

```diff
@@ -17,17 +17,16 @@ namespace TerminalApp
     enum Direction
     {
         None = 0,
+        Up,
+        Down,
         Left,
         Right,
-        Up,
-        Down
     };
 
     enum SplitState
     {
-        None = 0,
-        Vertical = 1,
-        Horizontal = 2
+        Vertical = 0,

```

Comment:
Should None/0 be omitted instead of renumbering these?

Reply:
Possibly, though I think most non-flags enums start with 0. Personally I'd just omit explicit numbering altogether.

Reply:
Sure, dump the numbering if you like.